### DISCUSSION TITLE
Posts about Granian 1.4.0

### DISCUSSION THREAD

**Discussion by tecbr** [emmett-framework/granian#304] [General]
I'm here again to say... Let Granian fly !

Many people still don't know Granian.

What do you think about posting about the **Granian 1.4.0** release on Python-related social media?

Reddit: [Python](https://www.reddit.com/r/python/), [LearnPython](https://www.reddit.com/r/learnpython/), [Django](https://www.reddit.com/r/django/), [Flask](https://www.reddit.com/r/flask/), [FastAPI](https://www.reddit.com/r/fastapi/), ...
HackerNews...

**Comment by gi0baro** [comment]
@tecbr I'm planning a blog post about 1.4, just need to find some time to properly do it.
In the meantime, feel free to share the new release wherever you prefer!

**Comment by gi0baro** [comment]
@tecbr here is the blog post in case you want to share it https://polar.sh/emmett-framework/posts/granian-1-4
